app/config.js

```diff
@@ -18,7 +19,8 @@ const schema = Joi.object({
 const config = {
   port: process.env.PORT,
   env: process.env.NODE_ENV,
-  cacheName: process.env.CACHE_NAME,
+  cacheName: 'redisCache',
```

Comment:
the cache name is only used within the application to give a name to a particular cache definition, so does not need defining externally so has been set to 'redisCache'